src/node_v8_platform-inl.h

```diff
@@ -99,6 +99,7 @@ struct V8Platform {
   }
 
   inline void Dispose() {
+    v8::V8::ShutdownPlatform();
```

Comment:
Should this come before or after the call to `StopTracingAgent()`? I see that `ShutdownPlatform()` calls `TracingCategoryObserver::TearDown()` so I would expect it to go after the call.

Reply:
I don’t think it makes a difference at this point, because it only removes a listener on the tracing controller (but does require that to be alive), and I do think this should be in reverse order of the setup code (otherwise something’s probably a bit off?)

Reply:
… I think CI proved you right :)